Kernel/Prekernel/multiboot.S

```diff
@@ -3,7 +3,7 @@
 .set MULTIBOOT_PAGE_ALIGN,    0x1
 .set MULTIBOOT_MEMORY_INFO,   0x2
 .set MULTIBOOT_VIDEO_MODE,    0x4
-.set multiboot_flags,         MULTIBOOT_PAGE_ALIGN | MULTIBOOT_MEMORY_INFO
+.set multiboot_flags,         MULTIBOOT_PAGE_ALIGN | MULTIBOOT_MEMORY_INFO | MULTIBOOT_VIDEO_MODE
```

Comment:
This is problematic to do - sometimes we don't want GUI on bare metal, so booting to 80x25 text console is desirable because of WindowServer bugs/breakage on bare metal...

Reply:
Dropped. On EFI GRUB allows setting video even without this flag. Not entirely up to spec but EFI often has no text mode available.